lightning/src/ln/functional_tests.rs

```diff
@@ -3905,6 +3905,120 @@ fn test_no_txn_manager_serialize_deserialize() {
 	send_payment(&nodes[0], &[&nodes[1]], 1000000, 1_000_000);
 }
 
+#[test]
+fn test_manager_serialize_deserialize_events() {
+	let chanmon_cfgs = create_chanmon_cfgs(2);
```

Comment:
Looks like the indentation is still off in this test. 

Also, could you add comments to each code block explaining what each part is doing?

Reply:
Oof, I thought I had taken care of the indentation stuff in my vimrc. I will take fix that and add in the comments.